pkgs/build-support/trivial-builders/test/default.nix

```
@@ -14,21 +14,24 @@
 { callPackage, lib, stdenv }:
 let
   inherit (lib) recurseIntoAttrs;
+  references = callPackage ./references {};
 in
 recurseIntoAttrs {
   concat = callPackage ./concat-test.nix {};
   linkFarm = callPackage ./link-farm.nix {};
   overriding = callPackage ../test-overriding.nix {};
+  # VM test not supported beyond linux yet
   references =
-    # VM test not supported beyond linux yet
     if stdenv.hostPlatform.isLinux
-    then callPackage ./references.nix {}
+    then references
     else null;
```

```suggestion
    else {};
```

It seems that this would please ofborg:
https://github.com/NixOS/nixpkgs/pull/204462/checks?check_run_id=19453355734